### DISCUSSION TITLE
Let's Encrypt

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [vercel/community#32] [Help] [answered]
Do you guys support `Let's Encrypt`

I wanna know if you support that service cus I seem to always depend on ssl and that's the only working service that adds ssl support

**Comment by paulogdm** [comment, marked as answer]
Hello,

Vercel will handle SSL for you automatically. You just need to add a domain to your project: https://vercel.com/docs/custom-domains

**Reply by holgerw** [reply]
I peaked at the certificate used by my Vercel hosted site and it seems Vercel uses Let's Encrypt under the hood.

Do you know if the Let's Encrypt changes in Jan 2021 that should result in certificate errors for Android users below Nougat (7.1.1) will be mitigated by Vercel? Seems that might affect up to 30% of Android devices out there.

https://letsencrypt.org/2020/11/06/own-two-feet.html

Heroku for example does not mitigate it transparently, but allows users to upload an alternative certificate.

https://help.heroku.com/LBVEYASY/acm-certificate-compatibility-faq

Can you comment on what we should expect from Vercel, please?
Thank you.

**Reply by oyedejioyewole** [reply]
I just wanted to know if they support Let's Encrypt, that's all

**Reply by leerob** [reply]
Yes, we use Let's Encrypt under the hood for SSL 👍
